src/components/QuickLink/QuickLink.test.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { axe, render, screen } from 'testUtils';
+
+import QuickLink from './';
+
+const text = 'links text';
+const href = '/';
+
+test('renders correctly', () => {
+  render(<QuickLink text={text} href={href} />);
+
+  expect(screen.getByText('links text')).toBeInTheDocument();
```

Comment:
```suggestion
  expect(screen.getByRole('link', { name: 'links text' })).toBeInTheDocument();
```
By default, use `getByRole` when possible following RTL's query priority - https://testing-library.com/docs/queries/about#priority (Same for line 29)

As a result, can also then kill the `renders the Flex component as an anchor tag` test